RFXtrx/__init__.py

```diff
@@ -303,6 +303,8 @@ def __init__(self, pkt):
         super(SensorEvent, self).__init__(device)
 
         self.values = {}
+        if isinstance(pkt, lowlevel.RfxMeter):
+            self.values['Value'] = pkt.value
```

Comment:
Maybe we can use the word 'counterValue', this is according to the protocol
description in the PDF.

On Wed, Aug 24, 2016 at 9:55 AM, Daniel Høyer Iversen <
notifications@github.com> wrote:

> In RFXtrx/__init__.py
> <https://github.com/Danielhiversen/pyRFXtrx/pull/29#discussion_r76009580>:
>
> > @@ -303,6 +303,8 @@ def __init__(self, pkt):
> >          super(SensorEvent, self).__init__(device)
> >
> >          self.values = {}
> > +        if isinstance(pkt, lowlevel.RfxMeter):
> > +            self.values['Value'] = pkt.value
>
> Is 'Value' the most descriptive word we can use?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Danielhiversen/pyRFXtrx/pull/29/files/f013abbd94e5c351791e96b34c3c10960558bccf#r76009580>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADD6VALVfn5pnxjijd7vf4jIVADpqUWhks5qi_jUgaJpZM4JrNqv>
> .
>